hack/verify-gofmt.sh

```diff
@@ -44,6 +44,7 @@ find_files() {
         -o -wholename '*/vendor/*' \
         -o -wholename './staging/src/k8s.io/client-go/*vendor/*' \
         -o -wholename '*/bindata.go' \
+        -o -wholename '*/zz_generated.openapi.go' \
```

Comment:
This can be removed once https://github.com/kubernetes/kube-openapi/pull/32 merges.

Reply:
It merged. Can you revendor?

Reply:
Bitbucket's down right now. Will try later https://status.bitbucket.org/

Reply:
🦄

Reply:
This explains why my dep experiments take ages today